flintrock/core.py

```diff
@@ -464,6 +475,20 @@ def generate_template_mapping(
         'spark_root_ephemeral_dirs': spark_ephemeral_dirs if spark_ephemeral_dirs else spark_root_dir,
     }
 
+    try:
+        template_mapping['region'] = cluster.region
+    except AttributeError:
+        logger.error('Property region is not defined in variable cluster')
+
+    # TODO: remove the following lines when the migration to IAM roles is finished
+    try:
+        import botocore
+        credentials = botocore.session.get_session().get_credentials()
+        template_mapping['access_key'] = credentials.access_key
+        template_mapping['secret_key'] = credentials.secret_key
+    except:
```

Comment:
`except AttributeError` + the ones that could be raised by botocore's get_session and get_credentials ?

Reply:
Since this code is temporary (we will soon remove it and use roles), and since trying to find exceptions that can be raised by get_session and get_credentials is a real pain, I would rather leave `except:`